src/DaedalusCompiler/Compilation/AssemblyBuilder.cs

```diff
@@ -292,6 +293,10 @@ public AssemblyBuildContext getEmptyBuildContext(bool isOperatorContext = false)
             };
         }
 
+        public bool isContextInsideFunction()
```

Comment:
please rename that to `isContextInsideExecBlock` case `active is FunctionBlock` is equel true as well for prototypes and instances

Reply:
Done